### ISSUE TITLE
Find a better name for the application

### ISSUE THREAD

**Issue by bzg** [codegouvfr/sill#189]
"SILL" is not a good name anymore.  Let's find a better name for a "software catalog" application.

**Comment by bzg** [comment]
Je propose "catalogi" : le pluriel latin de "catalogus" (catalogue) ou le génitif singulier ("du catalogue").

Je trouve ça générique et parlant.  WDYT?

**Comment by louvetv** [comment]
Est-ce qu'on a envie de garder la dimension libre des logiciels catalogués ?

**Comment by bzg** [comment]
> Est-ce qu'on a envie de garder la dimension libre des logiciels catalogués ?

Non, c'est une dimension nécessaire pour le SILL mais pas forcément pour tous les catalogues à venir.

L'indication de licence, quand elle est disponible, reste utile, même pour des licences non libres.

**Comment by bzg** [comment]
(Switching back to English to allow native English speakers to read and contribute.)

"Librothèque" sounds great, but it has two problems: it's French, and it focuses on free software, while the software aims to be more generic, not restricting admins to allowing only free software.

**Comment by laurentperrinet** [comment]
> (Switching back to English to allow native English speakers to read and contribute.)
> 
> "Librothèque" sounds great, but it has two problems: it's French, and it focuses on free software, while the software aims to be more generic, not restricting admins to allowing only free software.

ok, I better get the scope - too bad "logitech" is already taken 🤣

**Comment by inimod** [comment]
Hi all, can i suggest "SoftBox" ?

**Comment by bzg** [comment]
Yes, you can! I like "box" - I'm not sure "soft" is enough to connotate "software"... let's continue to brainstorm, we'll probably have a (consultative) vote in January.

**Comment by bzg** [comment]
I also suggest `logitheca`.

**Comment by bzg** [comment]
Well, thinking about this again, I suggest a more straightforward approach with "catalog".
